README.md

```diff
@@ -1,13 +1,27 @@
-# Ansible Devtools Demo EE
+# Ansible DevTools Execution Environment
+
+This is a container (execution environment) aimed toward being using
+for development and testing of ansible content. We should also mention that
+this container is not aimed to be used in production by Ansible users.
+
+It includes:
+
+* ansible
+* ansible-lint
+* molecule
+
+Among its main consumers we can mention [ansible-navigator](https://github.com/ansible/ansible-navigator) and [vscode-ansible](https://github.com/ansible/vscode-ansible) extension.
 
 ## Regenerating the build context with podman:
 
 ```bash
 $ tox -e podman
+...
 ```
 
 ## Regenerating the build context with docker:
 
 ```bash
```

Comment:
Since you're adding output, it's a `console` lexer should definitely be used.
```suggestion
```console
```